node/src/chain_spec.rs

```diff
@@ -316,6 +336,270 @@ fn manta_pc_testnet_genesis(
 				})
 				.collect(),
 		},
-		cumulus_pallet_aura_ext: Default::default(),
+		aura_ext: Default::default(),
+	}
+}
+
+// calamari chain specs
+#[cfg(feature = "calamari")]
+pub fn calamari_properties() -> Properties {
+	let mut p = Properties::new();
+	p.insert("ss58format".into(), constants::CALAMARI_SS58PREFIX.into());
+	p.insert("tokenDecimals".into(), constants::MANTA_DECIMAL.into());
+	p.insert(
+		"tokenSymbol".into(),
+		constants::CALAMARI_TOKEN_SYMBOL.into(),
+	);
+	p
+}
+
+#[cfg(feature = "calamari")]
+pub fn calamari_development_config(id: ParaId) -> CalamariChainSpec {
+	let properties = calamari_properties();
+
+	CalamariChainSpec::from_genesis(
+		// Name
+		"Calamari-Parachain Development",
+		// ID
+		"calamari_dev",
+		ChainType::Local,
+		move || {
+			calamari_dev_genesis(
+				// initial collators.
+				vec![(
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_collator_keys_from_seed("Alice"),
+				)],
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+				],
+				id,
+			)
+		},
+		vec![],
+		None,
+		Some(CALAMARI_PROTOCOL_ID),
+		Some(properties),
+		Extensions {
+			relay_chain: "polkadot-dev".into(),
```

Comment:
Maybe we can make the chain spec strings constants.

Reply:
Followed your comment.